README.md

```diff
@@ -1,24 +1,43 @@
+# Islandora Defaults    
+[![Build Status](https://travis-ci.com//Islandora/islandora_defaults.png?branch=8.x-1.x)](https://travis-ci.com//Islandora/islandora_defaults)
 [![Contribution Guidelines](http://img.shields.io/badge/CONTRIBUTING-Guidelines-blue.svg)](./CONTRIBUTING.md)
 [![LICENSE](https://img.shields.io/badge/license-GPLv2-blue.svg?style=flat-square)](./LICENSE)
 
 ## Introduction
 
-Islandora Defaults is a Drupal feature intended to showcase all of the functionality of a full Islandora install.
+Islandora Defaults is a Drupal feature intended to showcase all of the functionality of a full Islandora install. It provides custom content types, contexts, viewers, and other 'starter' configuration the reflects a baseline Islandora site. It is expected that you will take these defaults and customize them further to fit your use case. 
+
+## Installation
+Islandora_defaults is a Drupal Feature. See [Drupal documentation on Features](https://www.drupal.org/docs/8/modules/features) for more information.
+
+## Configuration
+Islandora_defaults is itself a set of configurations for Drupal. It can be further customized by customizing content types, actions, and other aspects of Drupal that are configured by islandora_defaults.
 
 ## Maintainers
 
 Current maintainers:
 
-* [Danny Lamb](https://github.com/dannylamb)
+* [Seth Shaw](https://github.com/seth-shaw-unlv)
 
-## Development
+## Documentation
+
+Further documentation for this module is available on the [Islandora 8 documentation site](https://islandora.github.io/documentation/).
+
+## Troubleshooting/Issues
 
-If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
+Having problems or solved a problem? Check out the Islandora google groups for a solution.
+
+* [Islandora Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora) 
+* [Islandora Dev Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora-dev)
+
+## Development
+If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora/documentation/wiki). We love to hear from you!
 
 If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
 
+We recommend using the [claw-playbook](https://github.com/Islandora-Devops/claw-playbook) to get started. If you want to pull down the submodules for development, don't forget to run `git submodule update --init --recursive` after cloning.
```

Comment:
I believe the 'If you want to pull down the submodules..." only applies to the project repository; not to islandora_defaults.

Reply:
Updated to remove that line.